### ISSUE TITLE
gulp-sass-error gives warning when used with gulp-sass 3.x

### ISSUE THREAD

**Issue by meganwalker-ibm** [timneutkens/gulp-sass-error#3]
gulp-sass is now at version 3.10, but the peerDependency in package.json is pegged to  `^2.3.2` causing an 'unmet peer dependency' warning to be issued if gulp-sass-error is used with gulp-sass 3.x

```
npm WARN gulp-sass-error@1.0.4 requires a peer of gulp-sass@^2.3.2 but none was installed.
```

It appears to work as intended still (from a quick sanity check rather than any formal verification), so I'd suggest upgrading the peer Dependency from `^2.3.2` to `>=2.3.2 <4.0.0` which will enable it for the 3.x range as well.

**Comment by timneutkens** [comment]
Feel free to send a pull request 👍

**Comment by meganwalker-ibm** [comment]
Thanks for submitting a PR for it - I've been busy for a couple of weeks.

**Comment by georgecrawford** [comment]
I've just run into this as well. I can also confirm things look good with gulp-sass 3.1.0.

**Comment by timneutkens** [comment]
Just published version `1.0.5` if anyone wants to co-maintain this project please let me know. As I don't use gulp-sass anymore :smile:
